vtex-apps/lightspeed
# [LIGHTSPD-34] fixed a bug where overnight sync was syncing with incorrect values

-Resolves the daily sync quantity misalignment issue
-Added splunk logging
-Added sync reset

How to verify
-Find an item that has different quantities in lightspeed and vtex
-Wait (or run the process-sync)
-Verify that the item quantity has been updated correctly on vtex
